spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkUtil.java

```diff
@@ -268,4 +279,59 @@ private static <T> List<T> toJavaList(Seq<T> seq) {
   private static String toExecutorLocation(BlockManagerId id) {
     return ExecutorCacheTaskLocation.apply(id.host(), id.executorId()).toString();
   }
+
+  /**
+   * Converts a value to pass into Spark from Iceberg's internal object model.
+   *
+   * @param type an Iceberg type
+   * @param value a value that is an instance of {@link Type.TypeID#javaClass()}
+   * @return the value converted for Spark
+   */
+  public static Object convertConstant(Type type, Object value) {
```

Comment:
I know that this is a copy of the other one, but I would also expect UUID to be here.

Reply:
Good point, I'll update it.

Reply:
The follow up will be here: https://github.com/apache/iceberg/pull/11811